kubernetes/apps/pipelines/geocoding/database/app/database.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: geocoding-database
+  namespace: geocoding
+spec:
+  imageName: ghcr.io/cloudnative-pg/postgis:16
+  enableSuperuserAccess: true
+  superuserSecret:
+    name: geocoding-database-superuser
+  bootstrap:
+    initdb:
+      postInitTemplateSQL:
+        - CREATE EXTENSION postgis;
+        - CREATE EXTENSION postgis_topology;
+        - CREATE EXTENSION fuzzystrmatch;
+        - CREATE EXTENSION postgis_tiger_geocoder;
+  postgresql:
+    parameters:
+      shared_buffers: "1GB"
+      work_mem: "50MB"
+      maintenance_work_mem: "10GB"
+      checkpoint_timeout: "300min"
+      max_wal_senders: "0"
+      wal_level: "minimal"
+      max_wal_size: "10GB"
+      checkpoint_completion_target: "0.9"
+      random_page_cost: "1.0"
+      full_page_writes: "off"
+      fsync: "off"
+  storage:
+    size: 100Gi
```

Comment:
Is this enough?

Reply:
Almost certainly not :laughing: But will be enough for now